auto_dev/data/repo/templates/autonomy/scripts/run_single_agent.sh

```diff
@@ -0,0 +1,152 @@
+#! /bin/bash
+set -e
+
+if [[ "$#" -lt 1 ]]; then
+    echo "Usage: $0 <author/agent_name> [--force] [--debug]"
+    echo "  <author/agent_name>: Identifier for the agent (e.g., eightballer/donation_station)"
+    echo "  --force            : Remove existing build directory if it exists"
+    echo "  --debug            : Run the agent in DEBUG mode"
+    exit 1
+fi
+
+# Initialize variables
+FORCE=false
+DEBUG=false
+AGENT_IDENTIFIER=""
+
+# Parse arguments
+while [[ "$#" -gt 0 ]]; do
+    case "$1" in
+        --force)
+            FORCE=true
+            ;;
+        --debug)
+            DEBUG=true
+            ;;
+        -*)
+            echo "Unknown option: $1"
+            echo "Usage: $0 <author/agent_name> [--force] [--debug]"
+            exit 1
+            ;;
+        *)
+            if [[ -z "$AGENT_IDENTIFIER" ]]; then
+                AGENT_IDENTIFIER="$1"
+            else
+                echo "Multiple agent identifiers provided."
+                echo "Usage: $0 <author/agent_name> [--force] [--debug]"
+                exit 1
+            fi
+            ;;
+    esac
+    shift
+done
+
+agent_author=$(echo "$AGENT_IDENTIFIER" | cut -d'/' -f1)
+agent_name=$(echo "$AGENT_IDENTIFIER" | cut -d'/' -f2)
+
+
+echo "   Agent author: $agent_author"
+echo "   Agent name:   $agent_name"
+
+CWD=$(pwd)
```

Comment:
Move all constants to the top so there's not a necessity to parse the whole file

Reply:
#370